docs/source/tutorials/multiannotator.ipynb

```diff
@@ -0,0 +1,753 @@
+    "\n",
+    "print(f\"Accuracy of 15 worst quality examples = {worst_quality_accuracy}\")\n",
+    "print(f\"Accuracy of better quality examples = {better_quality_accuracy}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4fdf4d91",
+   "metadata": {},
+   "source": [
+    "We observe that the 15 worst-consensus-quality-score examples have a lower average accuracy compared to the rest of the examples. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "06cae16a",
+   "metadata": {},
+   "source": [
+    "## 5. Retrain model using improved consensus labels"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8d4e31ab",
+   "metadata": {},
+   "source": [
+    "After obtaing the improved consensus labels, we can now retrain a machine learning model using these newly obtained labels. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6f8e6914",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = LogisticRegression()\n",
+    "\n",
+    "num_crossval_folds = 5  \n",
+    "improved_pred_probs = cross_val_predict(\n",
+    "    estimator=model, X=X, y=improved_consensus_label, cv=num_crossval_folds, method=\"predict_proba\"\n",
+    ")\n",
+    "\n",
+    "# alternatively, we can treat all the improved consensus labels as training labels to fit the model \n",
+    "# model.fit(X, improved_consensus_label)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e59f7d4f",
+   "metadata": {},
+   "source": [
+    "We can repeatedly iterate this whole process of getting better consensus labels using the model's out-of-sample predicted probabilites and then retraining the model with the improved labels to get even better predicted probabilities!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b806d2ea",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# Note: This cell is only for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
+    "\n",
+    "if majority_vote_accuracy >= cleanlab_label_accuracy:  # check cleanlab has improved prediction accuracy\n",
+    "    raise Exception(\"Cleanlab training failed to improve consensus label accuracy.\")\n",
+    "\n",
+    "annotator_stats = multiannotator_results[\"annotator_stats\"]\n",
+    "bad_annotator_idx = [\"A0046\", \"A0047\", \"A0048\", \"A0049\", \"A0050\"]\n",
+    "bad_annotator_mask = annotator_stats.index.isin(bad_annotator_idx)\n",
+    "\n",
+    "avg_annotator_quality_bad = np.mean(annotator_stats[bad_annotator_mask][\"annotator_quality\"])\n",
+    "avg_annotator_quality_good = np.mean(annotator_stats[~bad_annotator_mask][\"annotator_quality\"])\n",
+    "\n",
+    "if avg_annotator_quality_bad >= avg_annotator_quality_good:\n",
+    "    raise Exception(\"Low quality annotators have higher quality scores than good quality annotators\")"
+   ]
```

Comment:
Let's also have check of the consensus-quality score down here?
Eg. comparing `worst_quality_accuracy` against 
`better_quality_accuracy` and raising Exception if they don't compare as expected.